// Copyright 2015 gRPC authors.

syntax = "proto3";

package tests_of_grpc_testing;

message Up {
  int32 first_up_field = 1;
}

message Charm {
  int32 first_charm_field = 1;
}

message Top {
  int32 first_top_field = 1;
}
